decorate_test.go

```diff
@@ -164,7 +165,7 @@ func TestDecorateSuccess(t *testing.T) {
 			i := i
 			c.RequireProvide(func() *someInt { return newSomeInt(i) }, dig.Group("values"), dig.As(new(myInt)))
 		}
-		c.Decorate(func(in A) B {
+		c.RequireDecorate(func(in A) B {
```

Comment:
just noticed some yellow underlines in my IDE, i.e. missing err return handling.